server/delphilint-server/src/main/java/au/com/integradev/delphilint/remote/ImpactSeverity.java

```diff
@@ -22,7 +22,9 @@
 public enum ImpactSeverity {
   LOW,
   MEDIUM,
-  HIGH;
+  HIGH,
+  INFO,
```

Comment:
Should `INFO` be the 1st or the 4th value? (and in corresponding case statements too)

Reply:
Probably the first. I originally put it as the 4th everywhere, but that caused ordering issues when determining the maximum severity of an issue clientside.

Reply:
Done.